src/common/obj_bencher.cc

```diff
@@ -420,6 +420,7 @@ int ObjBencher::write_bench(int secondsToRun,
   bufferlist b_write;
   lock_cond lc(&lock);
   utime_t runtime;
+  runtime.set_from_double(secondsToRun);
```

Comment:
@Jing-Scott (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits